docs/tutorials/avro.ipynb

```diff
@@ -0,0 +1,379 @@
+{
```

Comment:
<p>Please create a section called <code>Setup</code> and mention the installation, import and package validation steps over there. </p><p> </p><p>Please refer: https://www.tensorflow.org/io/tutorials/elasticsearch#setup_packages</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/io/pull/1250/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>